src/index.d.ts

```diff
@@ -0,0 +1,23 @@
+export as namespace hyperappLogger
+
+import { ActionsType, View } from 'hyperapp'
+
+export interface LoggerFunction<PrevState = {}, Action = {}, NextState = {}> {
+  (prevState: PrevState, action: Action, nextState: NextState): void
+}
+
+export interface LoggerOptions {
+  log?: LoggerFunction
+}
+
+export interface App<State, Actions> {
```

Comment:
__Note:__ `App` interface will be added to `Hyperapp` typings in future https://github.com/hyperapp/hyperapp/pull/727

Reply:
Thanks, @joseluisq! Sorry, for the lack of response. I'm mostly focused on V2 at the moment and in my free time, which means I don't have a lot of time to look at your V1 PR. But feel free to leave comments indicating things such as this one and so on on that PR if need be — I'll read every one of them in due time. 👋

Reply:
Sure @jorgebucaran I can imagine.
I will keep tuned for news or reviews that are necessary.